app/views/tests/_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<h5>Nhóm sở thích S (Social): Xã hội</h5>
```

Comment:
Toàn bộ những phần text hiển thị phải được chuyển qua dùng I18n. Tham khảo: https://guides.rubyonrails.org/i18n.html
Optional: Tại sao phải I18n?

Reply:
I18n dùng để chuyển đổi ngôn ngữ cho trang web ạ